src/redux/watchList/reducer.js

```diff
@@ -4,52 +4,16 @@ import {
   DELETE_WATCHLIST,
   ADD_STOCK_TO_WATCHLIST,
   REPLACE_WATCHLIST_STOCKS,
+  FETCH_WATCHLIST_DETAIL_START,
+  FETCH_WATCHLIST_DETAIL_SUCCESS,
+  FETCH_WATCHLIST_DETAIL_FAIL,
 } from './actionType';
 
-const watchlist1 = {
-  id: 1,
-  name: 'A share',
-  stocks: [
-    {
-      id: 1,
-      ticker: 'MSFT',
-      currPrice: 141.21,
-      dailyChange: 0.23,
-      volume: '1.2M',
-    },
-    {
-      id: 2,
-      ticker: 'AMZN',
-      currPrice: 1208.2,
-      dailyChange: -0.06,
-      volume: '609k',
-    },
-    {
-      id: 3,
-      ticker: 'AAPL',
-      currPrice: 250.7,
-      dailyChange: 1.63,
-      volume: '810k',
-    },
-    {
-      id: 4,
-      ticker: 'FB',
-      currPrice: 87.3,
-      dailyChange: 0.81,
-      volume: '430k',
-    },
-  ],
-};
-
-const watchlist2 = {
-  id: 2,
-  name: 'NSDQ',
-  stocks: [],
-};
-
 const initialState = {
   counter: 3,
-  watchlists: [watchlist1, watchlist2],
+  watchlists: [],
+  loadingWatchlistDetails: false,
```

Comment:
be consequent, sometimes you say `fetchWatchlistDetail` somewhere it is `watchlistDetails` (singular-plural)